ui/app/AppLayouts/Communities/views/PermissionsView.qml

```diff
@@ -87,14 +88,24 @@ ColumnLayout {
         ]
     }
 
+    StatusListView {
+        id: listView
+        reuseItems: true
+        model: root.permissionsModel
+        spacing: 24
+        Layout.fillWidth: true
+        Layout.fillHeight: !useFullHeight
+        height: useFullHeight ? contentHeight : 0
```

Comment:
as the list in layout, we shouldn't bind to `height` (https://doc.qt.io/qt-6/qml-qtquick-layouts-layout.html#details). `Layout.preferredHeight` can be instead.